src/services/basicValidationMapping.js

```diff
@@ -0,0 +1,11 @@
+/** Mapping between basicValidationType and the basic validation login
+ * @param {Object} event - event triggered when a character is typed
+ */
+export const typeToFunctionMap = {
+    "numeric": function (event){
+        return (event.keyCode >= 48 && event.keyCode <= 57);
+    },
+    "alphanumeric": function(event){
+        return (event.keyCode >=48 && event.keyCode <= 57) || (event.keyCode >= 65 && event.keyCode <= 90) || (event.keyCode >= 97 && event.keyCode <= 122);
+    }
```

Comment:
Just a little refactoring can be done here!
Here, for every key, you're directly linking the function to that key.

A better way to write this would be to define two functions separately above this `export const ...` line.
These functions can have names like `isNumeric(input)` or `isAlphaNumeric(input)`. And then when those functions are defined and have a proper name, you can link those functions to they keys `numeric` and `alphanumeric` in the object that you're exporting.

Oh btw you will also be able to use `isNumeric` inside `isAlphaNumeric` function directly without repeating code :P